source/includes/fundamentals/code-snippets/Delete.java

```diff
@@ -52,35 +52,41 @@ public static void main(String [] args){
     }
 
     private void deleteManyExample(){
+        // Deletes all documents in the collection that contain a "qty" value of 0
         // begin deleteManyExample
         Bson filter = Filters.eq("qty", 0);
         collection.deleteMany(filter);
         // end deleteManyExample
     }
 
     private void findOneAndDeleteExample(){
+        // Deletes the first document with a "color" value of "purple" and prints the deleted document
         // begin findOneAndDeleteExample
         Bson filter = Filters.eq("color", purple);
```

Comment:
Issue:
Io think this code won't compile because "purple" isn't a defined symbol.

Suggestion:
While it's not a code comment, I think it would be good to correct this to the string "purple".